src/components/ui/PreviewProfile.jsx

```diff
@@ -0,0 +1,21 @@
+function PreviewProfile() {
+    return (
+        <div className='container relative :w-full h-full  my-5 flex justify-center'>
```

Comment:
Removed the fixed width of the parent container and replaced it with max-w-lg, which sets a maximum width of 32rem on smaller screens and allows the container to grow larger on larger screens.